PublicMapping/districtbuilder-classic
# Install district_builder_config Python package from separate repo

## Overview

Installs `district_builder_config` package from newly created GitHub repo: https://github.com/PublicMapping/district-builder-config

This keeps the configuration code separate and demonstrates there are no dependencies on Django. 

### Checklist


## Testing Instructions

This PR doesn't include any functional changes.

 * `docker-compose build`. The `district_builder_config` package should download and install successfully and the `generate_settings` script should also run successfully.
 * `docker-compose up` should work (app should run normally)
 * Run `docker-compose exec -T django ./manage.py setup -f config/config.xml` just for fun and see that it works (this demonstrates the setup command importing `StoredConfig` from the installed package correctly).
